Source Main 5.2/source/CComGem.cpp

```diff
@@ -34,7 +34,6 @@ namespace COMGEM
 
 void COMGEM::SendReqUnMix()
 {
-    iUnMixIndex += 12;
```

Comment:
No need to add this offset since we already added it when populating the list (for the dialog)